components/brave_rewards/browser/rewards_protocol_handler.cc

```diff
@@ -69,7 +69,7 @@ void LoadRewardsURL(
   const std::string bitflyer_staging_host = bitflyer_staging_url.host();
   const std::string gemini_staging_host = gemini_oauth_staging_url.host();
   const char* kAllowedDomains[] = {
-      "bitflyer.com",                 // bitFlyer production
+      "dummy.com",                    // bitFlyer production
```

Comment:
is it a real domain?

Reply:
No, the bitFlyer production domain is stubbed out for now per request from @Miyayes.

Reply:
@zenparsing rightly pointed out that this won't work as-is, so we're reverting it and will rely on stubbing the production client id in our builds instead.